.github/stale.yml

```diff
@@ -0,0 +1,18 @@
+# Number of days of inactivity before an Issue becomes stale
+daysUntilStale: 14
+# Number of days of inactivity before a stale Issue is closed
+daysUntilClose: 14
+# Only issues with all of these labels are checked if stale. Defaults to `[]` (disabled)
+onlyLabels:
+ - awaiting-user-response
+# Comment to post when marking as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as stale because it has no
+  recent activity. It will be closed if no further activity occurs. Thank you.
+# Comment to post when removing the stale label. Set to `false` to disable
+unmarkComment: false
+closeComment: >
+  Closing as stale. Please reopen if you'd like to work on this further.
+limitPerRun: 30
```

Comment:
What does `limitPerRun` mean? Can you put a comment here?

Reply:
I did a lot of helpful commenting in my version of this file https://github.com/bazelbuild/bazel/pull/14172/files

@rthadur I'd feel better as a ghosted contributor if you'd at least look through that and borrow the useful parts... 👻

Reply:
Haha , sure let me check , thank you